audio/soundtouch/Portfile

```diff
@@ -26,7 +27,11 @@ checksums           rmd160  57c876f09ddac0cdca51411705b92064e9da6d95 \
 use_autoreconf      yes
 autoreconf.args     -fvi
 
-configure.args      --disable-silent-rules \
```

Comment:
why are you removing this line, we want to keep it.

Reply:
Could have been a mistake in fact. Let me try and restore it.

Reply:
@reneeotten It is not removed, look below.